src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.AspNet.Identity;
+
+namespace Microsoft.Framework.Logging
```

Comment:
For example AppSignInManager returning custom AppSignInResult with an additional status. The user needs to log that